autorest/sender.go

```diff
@@ -283,7 +283,7 @@ func DoRetryForStatusCodesWithCap(attempts int, backoff, cap time.Duration, code
 func doRetryForStatusCodesImpl(s Sender, r *http.Request, count429 bool, attempts int, backoff, cap time.Duration, codes ...int) (resp *http.Response, err error) {
 	rr := NewRetriableRequest(r)
 	// Increment to add the first call (attempts denotes number of retries)
-	for attempt := 0; attempt < attempts+1; {
+	for attempt, delayCount := 0, 0; attempt < attempts+1; {
```

Comment:
What is the default or common value for attempts?

Reply:
The default is 3, specified here https://github.com/Azure/go-autorest/blob/master/autorest/client.go#L39 however it can be configured per client.